src/vmm/src/lib.rs

```diff
@@ -46,19 +46,19 @@ pub use config::*;
 mod devices;
 use devices::SerialWrapper;
 
-/// First address past 32 bits.
-const FIRST_ADDR_PAST_32BITS: u64 = 1 << 32;
+/// First address past 32 bits is where the MMIO gap ends.
+pub(crate) const MMIO_GAP_END: u64 = 1 << 32;
 /// Size of the MMIO gap.
-const MEM_32BIT_GAP_SIZE: u64 = 768 << 20;
-/// The start of the memory area reserved for MMIO devices.
-const MMIO_MEM_START: u64 = FIRST_ADDR_PAST_32BITS - MEM_32BIT_GAP_SIZE;
+pub(crate) const MMIO_GAP_SIZE: u64 = 768 << 20;
+/// The start of the MMIO gap (memory area reserved for MMIO devices).
+pub(crate) const MMIO_GAP_START: u64 = MMIO_GAP_END - MMIO_GAP_SIZE;
 /// Address of the zeropage, where Linux kernel boot parameters are written.
 const ZEROPG_START: u64 = 0x7000;
 /// Address where the kernel command line is written.
 const CMDLINE_START: u64 = 0x0002_0000;
 
-/// Default high memory start.
-pub const HIGH_RAM_START: u64 = 0x100000;
+/// Default high memory start (1 MiB).
+pub const HIGH_RAM_START: u64 = 0x0010_0000;
```

Comment:
nit: not necessarily related to the changes in this PR, but I would add the `DEFAULT` prefix to HIGH_RAM_START too, otherwise looks like the high memory always starts at this address.